entregas/lopezAndres/reto-001/DevolucionDinero.java

```diff
@@ -0,0 +1,37 @@
+
+public class DevolucionDinero {
+    public static void main(String [] args) {
+		int dineroPagado = 1000;
+		int dineroAPagar = 746;
+		int cambio = dineroPagado - dineroAPagar;
+		
+		int billetes100 = (int) cambio / 100;
+		cambio %= 100;
+		
+		int billetes50 = (int) cambio / 50;
+		cambio %= 50;
+		
+		int billetes20 = (int) cambio / 20;
+		cambio %= 20;
+		
+		int billetes10 = (int) cambio / 10;
+		cambio %= 10;
+		
+		int billetes5 = (int) cambio / 5;
+		cambio %= 5;
+		
+		int monedas2 = (int) cambio /2;
+		cambio %= 2;
+		
+		int monedas1 = (int) cambio /1;
```

Comment:
```suggestion
		int monedas1 = cambio;
```

O directamente usar cambio.